file_paths.md

```diff
@@ -305,6 +306,27 @@ The package will install a service named `puppetserver`, create a
         puppetserver/                     # writeable by puppetserver
             puppetserver.pid
 
+# puppetmaster
+
+The package will install a service named `puppetmaster`, create a
+`puppet` user and group, and run the service as the `puppet` user.
+
+    /opt/puppetlabs/server                # serverside apps live underneath
+        data
+            puppetmaster *                # :vardir (and $HOME for services that use it)
```

Comment:
nit, the package will create `/opt/puppetlabs/server` and `data` too, same question below?